packages/calypso-e2e/src/lib/utils/get-test-account.ts

```diff
@@ -0,0 +1,70 @@
+type Env = 'edge' | 'production';
+
+type SiteType = 'simple' | 'atomic';
+
+type DeepPartial< T > = T extends object
+	? {
+			[ P in keyof T ]?: DeepPartial< T[ P ] >;
+	  }
+	: T;
+
+type Plugin = 'gutenberg' | 'coblocks';
+
+type Accounts = { [ key in SiteType ]: { [ key in Env ]: string } };
+type AccountsIndex = { [ key in Plugin ]: Accounts };
+
+type OverridenAccounts = DeepPartial< Accounts >;
+
+type Options = {
```

Comment:
Perhaps this name can be a bit more explicit - something like `TestAccountOptions`?